src/storage-types/src/parameters.proto

```diff
@@ -18,7 +20,6 @@ import "service/src/params.proto";
 import "tracing/src/params.proto";
 
 message ProtoStorageParameters {
-  reserved 1, 2, 3, 25, 12, 16, 26, 31, 39;
```

Comment:
We could re-use the small identifiers, they have a more efficient proto encoding.

Reply:
Good point, might as well clean up the ids here too.

Reply:
I've done that and also reordered the proto fields to match the order in Rust.